torch/_subclasses/meta_utils.py

```diff
@@ -756,10 +757,11 @@ def meta_tensor(
         self,
         t: MetaTensorDesc,
         shape_env: Optional[ShapeEnv],
-        callback: Callable[[Callable[[], torch.Tensor]], _TensorT],
+        callback_: Callable[[Callable[[], torch.Tensor]], _TensorT],
```

Comment:
maybe a dumb nit but any reason not to keep the arg name here the same? (and do `callback = functools.partial(callback, device=t.device)`). I agree this isn't public API though

Reply:
Yep you're right `callback = functools.partial(callback, device=t.device)` should just overwrite it